src/mii_sequences/elias_fano.rs

```diff
@@ -691,10 +699,6 @@ mod tests {
     #[test]
     fn test_from_bits_empty() {
         let e = EliasFano::from_bits([]);
-        assert_eq!(
-            e.err().map(|x| x.to_string()),
-            Some("bits must not be empty.".to_string())
-        );
```

Comment:
The assert_eq should not be omitted because it is expected that the error value is returned.

Reply:
Sorry for making extra work for you, this was again a point where I thought the error would not occur anymore.